File: general-client/src/test/java/com/song/middleware/client/util/InetUtilsTest.java
package com.song.middleware.client.util;

import java.net.InetAddress;
import org.junit.Test;

/**
 * Created by song on 2017/8/6.
 */
public class InetUtilsTest {

    @Test
    public void findFirstNonLoopbackAddress() throws Exception {
        InetAddress address = InetUtils.findFirstNonLoopbackAddress();
        System.out.println(address.getHostAddress());
        System.out.println(address.getHostName());
    }

}

File: general-client/src/test/java/com/song/middleware/client/util/InetUtilsTest.kt
package com.song.middleware.client.util

import org.junit.Test

/**
 * Created by song on 2017/8/6.
 */
class InetUtilsTest {

    @Test
    @Throws(Exception::class)
    fun findFirstNonLoopbackAddress() {
        val address = InetUtils.findFirstNonLoopbackAddress()
        println(address!!.hostAddress)
        println(address.hostName)
    }

}